secondservice/src/main/java/com/services/secondservice/service/UserService.java

```diff
@@ -0,0 +1,36 @@
+package com.services.secondservice.service;
+
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.RestTemplate;
+
+import com.services.secondservice.db.entity.user.User;
+import com.services.secondservice.db.repositories.user.UserRepository;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+
+@Service
+@RequiredArgsConstructor
+@Slf4j
+public class UserService {
+
+    private final UserRepository userRepository;
+    private final RestTemplate restTemplate;
+
+    public User create(User user) {
+        HttpEntity<User> entity = new HttpEntity<User>(user);
+        ResponseEntity<User> responseEntity = restTemplate.exchange("http://thirdservice:8083/user", HttpMethod.POST,
+                entity, User.class);
+        User createdUser = responseEntity.getBody();
+        createdUser.setMiddleName(user.getMiddleName());
+        log.info("Get user: {}", createdUser);
+        if (responseEntity.getStatusCode().is2xxSuccessful()) {
+            return userRepository.save(createdUser);
+        } else {
+            return null;
```

Comment:
Consider throwing an exception here and handling it in the controller. Otherwise, the user will just receive `null` with `201 CREATED` status. Same for the other methods (and services). 

You should throw an exception if something goes wrong, then handle it in the controller and set response code accordingly (i.e. `404 NOT FOUND` if it's `GET` method).